packages/webview_flutter/ios/Classes/WebViewFlutterPlugin.m

```diff
@@ -7,6 +8,7 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
   FLTWebViewFactory* webviewFactory =
       [[FLTWebViewFactory alloc] initWithMessenger:registrar.messenger];
   [registrar registerViewFactory:webviewFactory withId:@"plugins.flutter.io/webview"];
+  FLTCookieManager* cookieManager = [[FLTCookieManager alloc] init:registrar.messenger];
```

Comment:
What's keeping a reference to this instance?

Reply:
the instance is not being used, but the init method needs to be called. Removing the return reference.

Reply:
If we don't keep a reference what prevents FLTCookieManager from being disposed by the time we got a method channel invocation?
(we are running this with ARC)